### ISSUE TITLE
Migration of large test suite

### ISSUE THREAD

**Issue by pesche666** [ingwarsw/arquillian-suite-extension#35]
Hi 

I try to migrate a larger test suite of 300 Arquillian Test Classes (each with their own micro deployment)   to migrate to suite extension approach, with much fewer deployments. I would like migrate this incrementally, class by class. However the migration path you suggest would be quite awkward for me since you need to move all three hundred deployments to one class and annotate each test method. Essentially you have to modify each class and each test. What I would like to do is create one deployment, then move some test classes to this deployment and see how it works.

Basically it would be great if a test class would use its own @Deployment method if it has one, otherwise it would use the suite @Deployment. Can this somehow be achieved?

Thanks!

**Comment by ingwarsw** [comment]
I was thinking about that kind of solution but I couldn't make it possible without huge effort.

Guys from arquillian are working on supporting that out of the box but.. 

Here is discussion: http://discuss.arquillian.org/t/arquillian-core-suite-subsuite-how-to-define-the-suite/163/28

Work is in progress..
https://github.com/aslakknutsen/arquillian-core/tree/arq_suite

**Comment by hoeppy** [comment]
Maybe you can use the EmbeddedGradleImporter (if you build with gradle) or MavenResolver to create a Deployment of your whole application. Then you dont have to merge all micro diployments.
Than comment the deployment method of each test.
Most of the test should run and then you only have to look at failed one.
That save some time for me
